Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/httpbody/UserResponse.java

```diff
@@ -21,10 +23,22 @@
     @Expose
     private String username;
 
+    @SerializedName("profile_picture")
+    @Expose
+    private String image;
```

Comment:
Name this field as profilePicture so that it is consistent with remote name. Otherwise, it becomes hard to keep track of which field corresponds to which JSON field.